### PR TITLE
Add basic information to xrrenderstate.md.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: webxr-device-api/xrrenderstate.md [hunk 1]
```diff
@@ -6,19 +6,19 @@ The **`XRRenderState`** interface of the WebXR API provides information about th
...
-  <dd>TBD</dd>
+  <dd>The distance in meters of the near clip plane from the viewer.</dd>
 
   <dt>inlineVerticalFiledOfView \{\{readonlyinline\}\}</dt>
-  <dd>TBD</dd>
+  <dd>Defines the angle of the field of view in radians used when computing projection matrices for an `inline` `XRSession` objects. This option must be `null` for immersive sessions.</dd>
 
   <dt>outputContext \{\{readonlyinline\}\}</dt>
-  <dd>TBD</dd>
+  <dd>An <code><a href="xrlayer">XRLayer</a></code> instance, which is a wrapper object for the \{\{domxref("HTMLCanvasElement")\}\} object on which AR or VR content will be drawn. To retrieve an instance of this object, call `getContext()` on a canvas object with the value `'xrpresent'`.</dd>
```
Comment (toji) [inline webxr-device-api/xrrenderstate.md]: Two notes: One is that this option will be removed very soon, and in fact is not included in the Chrome 76 OT. Two is that when it was available it did not take an XRLayer or an HTMLCanvasElement instance, but an XRPresentationContext (which was retrieved from an HTMLCanvasElement with `.getContext()`)

Comment (jpmedley) [reply webxr-device-api/xrrenderstate.md]: I removed `outputContext'. PTAL

Comment (jpmedley) [reply webxr-device-api/xrrenderstate.md]: Here: https://github.com/immersive-web/webxr-reference/pull/25/commits/d93029366390b0265c952629f395d37ba137c1a2

**Comment by toji** [comment]
LGTM! Feel free to merge at your leisure.

**Pull request merged**